### ISSUE TITLE
Make a Criteria for Code Example!! 

### ISSUE THREAD

**Issue by Suzan009** [keras-team/keras-io#640]
It's nice to see [this code example](https://keras.io/examples/vision/), for example in computer vision. But it also looks like it doesn't follow any acceptance rules and randomly accepting the code examples. 

### The same type of code example should not be accepted separately. The examples should be **general** for the most part. For example, the following code example can be merged into one single code example. There is no need to make a part. It's redundant and makes the page less professional. 

(1).
- [CutMix data augmentation for image classification](https://keras.io/examples/vision/cutmix/)
- [MixUp augmentation for image classification](https://keras.io/examples/vision/mixup/)
- [RandAugment for Image Classification for Improved Robustness](https://keras.io/examples/vision/randaugment/)

(2). 
- [Image similarity estimation using a Siamese Network with a contrastive loss](https://keras.io/examples/vision/siamese_contrastive/)
- [Image similarity estimation using a Siamese Network with a triplet loss](https://keras.io/examples/vision/siamese_network/)

# Code example should be general. 

- [Image segmentation with a U-Net-like architecture](https://keras.io/examples/vision/oxford_pets_image_segmentation/)
- [Multiclass semantic segmentation using DeepLabV3+](https://keras.io/examples/vision/deeplabv3_plus/)

It's better to have one example where single/multiple class segmentation is demonstrated. In the above example, the second example should be updated. It's simply showing the implementation of DeepLabV3+ only which can be found in other repo. It doesn't even explain the data set (eda). 

# Make it more useful 

- [Gradient Centralization for Better Training Performance](https://keras.io/examples/vision/gradient_centralization/)
- [Involutional neural networks
](https://keras.io/examples/vision/involution/)

It's quite surprising how these two kernel got excepted. It's simply just implementation which also can be found by googling. It should demonstrate the engineering solution by this approach in detail, but sadly it doesn't. 

# Good Code Example 

Here is the list of very good code examples, just for references:

- [OCR model for reading Captchas](https://keras.io/examples/vision/captcha_ocr/)
- [Image Captioning](https://keras.io/examples/vision/image_captioning/)
- [Object Detection with RetinaNet](https://keras.io/examples/vision/retinanet/)
- [3D image classification from CT scans](https://keras.io/examples/vision/3D_image_classification/)
- [Learning to Resize in Computer Vision](https://keras.io/examples/vision/learnable_resizer/)
- [Image segmentation with a U-Net-like architecture](https://keras.io/examples/vision/oxford_pets_image_segmentation/)
- [Handwriting recognition](https://keras.io/examples/vision/handwriting_recognition/)
- [Image classification via fine-tuning with EfficientNet](https://keras.io/examples/vision/image_classification_efficientnet_fine_tuning/)

**Comment by Suzan009** [comment]
cc. @fchollet
I don't mean to disrespect the example authors but the criteria for code example has to be fixed.

**Comment by innat** [comment]
@fchollet I partially agree with @Suzan009. Rule no. 3 isn't followed properly. **IMHO**, for the high-quality code example, it may be considered and can be added. But in general, the contributor should be aware of the submission rules clearly to avoid topic repetition. 

![ASA](https://user-images.githubusercontent.com/17668390/133982431-1e14968b-701c-42c6-948e-bc09d5f8b3eb.png)

**Comment by innat** [comment]
@fchollet 
Or, the code example can be categorized as something similar to [paper-with-code](https://paperswithcode.com/sota). (just sharing an idea).
